tests/hooks/useSynchronizedRequest.test.tsx

```diff
@@ -1,5 +1,5 @@
 import { useSynchronizedRequest } from '../../src/hooks/useSynchronizedRequest';
-import { renderHook } from '@testing-library/react-hooks';
+import { renderHook } from '@testing-library/react-hooks/dom';
```

Comment:
how come we need `/dom`?

Reply:
`/dom` is needed to resolve the warning:
`Warning: It looks like you're using the wrong act() around your test interactions.
Be sure to use the matching version of act() corresponding to your renderer:

// for react-dom:
import {act} from 'react-dom/test-utils';
// ...
act(() => ...);

// for react-test-renderer:
import TestRenderer from react-test-renderer';`

Reply:
I checked with a couple other Slapshot folks, we weren't able to see this warning. It may be related to your NPM/Node version. I'd rather we avoid using the `dom` version of `renderHook` if possible. Depending on the `dom` version can lead to other, nuanced issues.